packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -164,7 +164,7 @@ export default async function processDeployment({
         printInspectUrl(output, event.payload.inspectorUrl, deployStamp);
 
         if (quiet) {
-          process.stdout.write(`https://${event.payload.url}`);
+          process.stdout.write(`https://${event.payload.url}\n`);
```

Comment:
I don't think this is a good idea. The deployment only prints the url to stdout. It shouldn't print anything else to stdout.